src/ansys/fluent/core/utils/search.py

```diff
@@ -21,12 +33,36 @@
     Workflow,
 )
 
+_THIS_DIRNAME = os.path.dirname(__file__)
+
+
+class ValueConflict(ValueError):
+    """Raised when both ``wildcard`` and ``match_whole_word`` are ``True``."""
 
-def get_api_tree_file_name(version: str) -> Path:
+    def __init__(self):
+        super().__init__(
+            "Provide either ``wildcard`` or ``match_whole_word`` as ``True``."
+        )
+
+
+def get_api_tree_file_name(
+    version: Optional[str] = None,
```

Comment:
`version` is the first argument but the last resort.

Reply:
Right, we have changed the order now. Thank you.